src/hu_nmt/data_augmentator/graph_mappers/ged.py

```diff
@@ -0,0 +1,92 @@
+import networkx as nx
+from networkx import graph_edit_distance
+from typing import Dict
+
+from hu_nmt.data_augmentator.dependency_parsers.dependency_parser_factory import DependencyParserFactory
+from hu_nmt.data_augmentator.graph_mappers.graph_similarity_base import GraphSimilarityBase
+
+
+class GED(GraphSimilarityBase):
```

Comment:
Can you add a docstring explaining the main logic and motivation between this class as well please?